incubator/ethereum/templates/geth-account.secret.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: {{ template "ethereum.fullname" . }}-geth-account-secret
+  labels:
+    app: {{ template "ethereum.name" . }}
+    chart: {{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+    component: geth-account-secret
```

Comment:
It's not a component of its own. I'd remove the label.

Reply:
Fixed.